safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -17,8 +17,13 @@
 
 from shutil import copy
 from PyQt4 import QtCore
-from PyQt4.QtCore import QVariant
-from PyQt4.QtNetwork import QNetworkReply
+from PyQt4.QtScript import QScriptEngine, QScriptValueIterator
```

Comment:
Comment your block of code and remove this line I think. I don't have a strong opinion